code/modules/mob/living/carbon/human/species/diona.dm

```diff
@@ -70,16 +72,43 @@
 		return 1
 	return 0
 
+// Only roundstart and arriving dionaea should have had time to prepare a seed pod. 
+// Terrarium and Pod dionaea will likely have access to botany and won't need this ability. 
+/datum/species/diona/after_equip_job(datum/job/J, mob/living/carbon/human/H)
```

Comment:
This is hacky at heck to do this.

Reply:
Is this a bit closer to alright?

Reply:
It would be more helpful if you told them why it’s hacky and what they should do or use instead